pkg/allocator/allocator.go

```diff
@@ -155,7 +155,21 @@ type Allocator struct {
 type AllocatorOption func(*Allocator)
 
 // NewAllocatorForGC returns an allocator that can be used to run RunGC()
-func NewAllocatorForGC(backend Backend) *Allocator {
+//
+// The allocator can be configured by passing in additional options:
+//  - WithMin(id) - minimum ID to allocate (default: 1)
+//  - WithMax(id) - maximum ID to allocate (default max(uint64))
+func NewAllocatorForGC(backend Backend, opts ...AllocatorOption) *Allocator {
```

Comment:
I guess we can prevent this from ever happening again by making the constructors take the min and max, but maybe that's not useful anymore now. Good catch.

Reply:
Aditi above slightly touched on this.